Units/parser-vala.r/class.vala.b/expected.tags

```diff
@@ -0,0 +1,13 @@
+main	input.vala	/^void main(string[] args) {$/;"	method	line:8	language:Vala
+Address	input.vala	/^class Address {$/;"	class	line:15	language:Vala
+Person	input.vala	/^class Person {$/;"	class	line:23	language:Vala
+Address	input.vala	/^   public Address address {get; set;}$/;"	field	line:24	language:Vala
+_name	input.vala	16;"	field	line:16	language:Vala
+d_age	input.vala	17;"	field	line:17	language:Vala
+name	input.vala	16;"	property	line:16	language:Vala
```

Comment:
@masatake this is a property.

class Person {
   public string name {get; set;}
}

at same time it is a field. In vala you can do something like

class Person {
   private int d_age;

   public int age {
      get { return d_age;}
      set { 
         if (value > 0) {
            d_age = value;
         } else {
            d_age = 0;
         }
      }
   }
}

`d_age` is a private field and `age` is the property.

Old implementation use `_<fieldname>` when there's no private field. Not sure if that really makes sense (pseudo tag or virtual tag?)

In general a field is:

<scope>* <type> <name>

if there's no scope it is private.

A property (can be more complex but is:

field {get; set;}

A property notifies its changes so you can add a listener, force to fill on constructor... things like that.

Reply:
No indentation breaks my small brain. Could you insert proper whitespaces?